ckan/logic/action/update.py

```diff
@@ -307,10 +308,11 @@ def package_update(context, data_dict):
     else:
         rev.message = _(u'REST API: Update object %s') % data.get("name")
 
-    #avoid revisioning by updating directly
-    model.Session.query(model.Package).filter_by(id=pkg.id).update(
-        {"metadata_modified": datetime.datetime.utcnow()})
-    model.Session.refresh(pkg)
+    if not context.get('resources_only', False):
```

Comment:
should this be 
if not context.get('resources_only', True):

since it does not exist. and is never false or does context.get return none which triggers this?